spec/models/custom_attribute_spec.rb

```diff
@@ -12,6 +12,9 @@
   end
 
   it "returns the value type of Fixnum custom attributes" do
-    expect(int_custom_attribute.value_type).to eq(:fixnum)
+    # TODO: Ruby 2.4 unifies fixnum and bignum into integer, we shouldn't be
+    # returnings ruby types like this.
+    expected = RUBY_VERSION >= "2.4.0" ? :integer : :fixnum
+    expect(int_custom_attribute.value_type).to eq(expected)
```

Comment:
To be safe @cben, others, I kept this test but made it return what ruby 2.4.0 is expecting.  We'll have to address this as we move to ruby 2.4.0 more.  This should be enough to get us compatible for now.